model-optimizer/extensions/back/add_outputs_recursive.py

```diff
@@ -27,6 +27,26 @@ def ti_set_output_port_shape(cycle_node, internal_id, port_num, iterations_count
     cycle_node.out_port(port_num).data.set_shape(out_shape)
 
 
+def get_iterations_count_from_output_record(output_rec):
+    def check_field(record, field):
+        return field in record and record[field] is not None
+
+    iterations_count = 1
+    # 1. check if we need concatenate iteration results for given output
+    if not check_field(output_rec, 'axis'):
+        # in this case we dont concat outputs, so iterations count is not needed really
+        return iterations_count
```

Comment:
why in this case do we return `1`? Logically, it should be `None`

Reply:
agree